vlrouter.go

```diff
@@ -64,18 +64,20 @@ type Vlrouter struct {
 	anycastMac    net.HardwareAddr   //Anycast mac used for local endpoints
 	myBgpPeer     string             // bgp neighbor
 	unresolvedEPs cmap.ConcurrentMap // unresolved endpoint map
+	uplinkOfp     uint32             // uplink intf portno mapping
+
 }
 
 // GetUplink API gets the uplink port with uplinkID from uplink DB
-func (self *Vlrouter) GetUplink(uplinkID string) *PortInfo {
-	uplink, ok := self.uplinkPortDb.Get(uplinkID)
+func (vl *Vlrouter) GetUplink(uplinkID string) *PortInfo {
```

Comment:
Minor nit: can we skip this change or make a separate commit in the PR? It's taking the focus off the actual changes in the PR.